test/test_ff_utils.py

```diff
@@ -603,6 +603,146 @@ def test_expand_es_metadata(integrated_ff):
     assert test_item['lab'].startswith('828cd4fe')
 
 
+@pytest.mark.integrated
+def test_get_item_facets(integrated_ff):
+    """ Tests that we can resolve information on facets """
+    key, ff_env = integrated_ff['ff_key'], integrated_ff['ff_env']
+    item_type = 'experiment_set_replicate'
+    facets = ff_utils.get_item_facets(item_type, key=key, ff_env=ff_env)
+    assert 'Lab' in facets
+    assert 'Center' in facets
+    assert 'Set Type' in facets
+    assert 'Internal Release Date' in facets
+
+
+@pytest.mark.integrated
+def test_search_facets_exp_set(integrated_ff):
+    """ Tests the experiment set search features using mastertest """
+    key, ff_env = integrated_ff['ff_key'], integrated_ff['ff_env']
+    url = integrated_ff['ff_key']['server']
+    all_facets = ff_utils.get_item_facets('experiment_set_replicate', key=key, ff_env=ff_env)
+    for_all = {'base_url': url, 'key': key, 'ff_env': ff_env, 'item_facets': all_facets}
+    project = {'Project': '4DN'}
+    project.update(for_all)
+    resp = ff_utils.search_facets(kwargs=project)
+    assert len(resp) == 8
```

Comment:
Rather than testing a specific number of results, it might be better to test that there any results and make sure the results all have the expected field=value